### PR TITLE
Show books by category

### PR DESCRIPTION
**Pull request by hoangtv2605**
## Related Tickets:
- [26967](https://edu-redmine.sun-asterisk.vn/issues/26967)

## WHAT this PR does?
- Show books by category

## Notes Impacted Areas:
- N/A.

### CODE REVIEW

**Comment by hoangtv2605** [comment]
ready

File: resources/views/book.blade.php [hunk 1]
```diff
@@ -16,38 +16,73 @@
...
+            <div class="col-md-3 products-left">
+                <div class="css-treeview">
+                    <h4>{{ trans('message.categories') }}</h4>
+                    <table id="tree-table" class="table table-hover table-bordered mb-0">
+                        <tbody>
+                            @foreach ($parentCategories as $category)
+                                <tr data-id="{{$category->id}}" data-parent="0" data-level="1">
+                                    <td data-column="name">
+                                        <a class="decoration-none" href="{{ route('show_by_category', [$category->name, $category->id]) }}">
+                                            {{$category->name}}
```
Comment (huyenttm-0901) [inline resources/views/book.blade.php]: convention

File: resources/views/book_category.blade.php [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            <div class="col-md-3 products-left">
+                <div class="css-treeview">
+                    <h4>{{ trans('message.categories') }}</h4>
+                    <table id="tree-table" class="table table-hover table-bordered mb-0">
+                        <tbody>
+                            @foreach ($parentCategories as $category)
+                                <tr data-id="{{$category->id}}" data-parent="0" data-level="1">
+                                    <td data-column="name">
+                                        <a class="decoration-none" href="{{ route('show_by_category', [$category->name, $category->id]) }}">
+                                            {{$category->name}}
```
Comment (huyenttm-0901) [inline resources/views/book_category.blade.php]: convention

File: resources/views/sub_categories_list.blade.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@foreach ($subCategories as $subCategory)
+    <tr data-id="{{$subCategory->id}}" data-parent="{{$dataParent}}" data-level = "{{$dataLevel + 1}}">
+        <td data-column="name">
+            <a class="decoration-none text-green" href="{{ route('show_by_category', [$subCategory->name, $subCategory->id]) }}">
+                {{$subCategory->name}}
```
Comment (huyenttm-0901) [inline resources/views/sub_categories_list.blade.php]: convention

**Review by huyenttm-0901 — COMMENTED** [review]
thêm link ticket

**Comment by hoangtv2605** [comment]
ready

**Comment by hoangtv2605** [comment]
ready

File: resources/views/sub_categories_list.blade.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@foreach ($subCategories as $subCategory)
+    <tr data-id="{{ $subCategory->id }}" data-parent="{{$dataParent}}" data-level = "{{$dataLevel + 1}}">
```
Comment (huyenttm-0901) [inline resources/views/sub_categories_list.blade.php:2]: convention

**Review by huyenttm-0901 — COMMENTED** [review]
còn convention, fix ở pull sau

**Pull request merged**